docker/docker-compose-mysql-cs8.yml

```diff
@@ -8,7 +8,7 @@ services:
     volumes:
         - ./mysql.cnf:/etc/mysql/conf.d/mysql.cnf
   candlepin:
-    image: ${REGISTRY}/candlepin-base
+    image: ${REGISTRY}/candlepin-base-cs8
```

Comment:
Wouldn't this change be enough to make this work? Why the rest?

Reply:
yes, but i was thinking if we decide in future to run different OS like with postgres, so I thought it would be ready.